crates/rpc/types/block_identifier.rs

```diff
@@ -81,19 +81,6 @@ impl BlockIdentifierOrHash {
         }
     }
 
-    pub fn is_latest(&self, storage: &Store) -> Result<bool, StoreError> {
```

Comment:
do we want to keep this for future use maybe?

Reply:
Clippy wanted it gone, but we can add #[allow(unused)] if we want to keep it

Reply:
Let's keep it for now